### ISSUE TITLE
Latest release of tsh URL.

### ISSUE THREAD

**Issue by russjones** [gravitational/teleport#2777]
For automation purposes, it would be nice to have a latest link symlink can be used to download the latest version of tsh.

For example https://get.gravitational.com/latest.tar.gz -> https://get.gravitational.com/teleport-v3.2.6-darwin-amd64-bin.tar.gz

**Comment by benarent** [comment]
It would also be handy to add this to the Quay tags https://quay.io/repository/gravitational/teleport?tab=tags

**Comment by knisbet** [comment]
Something else to think about, is whether it should be a universal latest, or whether it should be broken down by cycle. The latest 4.0 release, the latest LTS release, latest beta, etc.

**Comment by t-lark** [comment]
Yes, any URL we can hit for the `latest release` and have it take us to the latest release would be very much appreciated

**Comment by stmuraka** [comment]
perhaps just metadata/tag reference to the `latest`? when we download, we also would need to know the version we're downloading - not just a blind download. perhaps something we can query for the latest version.

**Comment by wenksi** [comment]
Would simplify Teleport automation with Ansible.

**Comment by webvictim** [comment]
Although not a solution to this exact issue, I thought I’d point out that we have recently added a new set of nightly-rebuilt Docker images.

These images will point to the latest version of Teleport from the three most recent release branches.

For example:

`quay.io/gravitational/teleport:4.3` points to the latest 4.3 OSS release (currently 4.3.2)
`quay.io/gravitational/teleport-ent:4.2` points to the latest 4.2 Enterprise release (currently 4.2.11)
`quay.io/gravitational/teleport-ent:4.1-fips` points to the latest 4.1 Enterprise FIPS release (currently 4.1.10)

So if you want to always be running the latest version of Teleport on a given branch, pulling these images regularly and restarting your Teleport container can help
with that.

**Comment by maddeye** [comment]
any update or furture plans on this?
